x-pack/plugin/inference/src/yamlRestTest/resources/rest-api-spec/test/inference/10_semantic_text_field_mapping.yml

```diff
@@ -208,6 +266,57 @@ setup:
 
   - match: { fields.dense_field.semantic_text.searchable: true }
 
+---
+"Field caps set include_empty_field to true with text embedding":
+
+  - requires:
+      cluster_features: "gte_v8.16.0"
+      reason: field_caps support for semantic_text added in 8.16.0
+
+  - do:
+      field_caps:
+        include_empty_fields: true
+        index: test-index
+        fields: "*"
+
+  - match: { indices: [ "test-index" ] }
+  - is_true: fields.sparse_field
+  - is_true: fields.dense_field
```

Comment:
You can integrate this check into the previous test, just like with the sparse embedding test